packages/hub/utils/queries.ts

```diff
@@ -9,10 +9,17 @@ const camelToSnakeCase = (str: string) => str.replace(/[A-Z]/g, (letter) => `_${
 // => { where: 'name=$1 AND age=$2', values: [ 'Phil', 45 ] }
 
 export function buildConditions(params: any) {
-  let nonNullParams = pickBy(params, function (value) {
+  // Only allow nulls, strings and numbers in the params
+  let filteredParams = Object.fromEntries(
```

Comment:
Preparation for https://github.com/cardstack/cardstack/pull/2594/commits/d8abeab3f06c534f46d99cbc0a55800d8a9efaeb#diff-4dc024baa8514b72c704540caefc15706cd8e19273678717f4d573381aa7b6c0R49